test/library/standard/Types/isNothing-proc.chpl

```diff
@@ -0,0 +1,32 @@
+use Types;
+
+proc isNothing(type t)  param return isNothingType(t);
+proc isNothing(e)   param return isNothingType(e.type);
```

Comment:
These functions should not be in the test since we want to test the ones in Types